src/devices/Amg88xx/StatusClearBit.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Amg88xx
+{
+    /// <summary>
+    /// Defines the status bits of the status clear register (addr: 0x05)
+    /// </summary>
+    [Flags]
+    public enum StatusClearBit : byte
```

Comment:
same as everywhere for the enums, I don't think it should be exposed as you have methods for that. If it needs to be exposed, then use proper naming convention

Reply:
Can be internal. I use the InternalsVisibleToAttribute to expose it to the unit test assembly.